acceptance_tests/features/new_case.feature

```diff
@@ -0,0 +1,6 @@
+Feature: A new case is submitted
+
+ Scenario: A new case is submitted
+   Given sample file "sis_survey_link.csv" with sensitive columns [PHONE_NUMBER,CHILD_NAME] is loaded successfully
+   When a newCase event is built and submitted
+   Then the event logged against the case is [NEW_CASE]
```

Comment:
I think it should check the case update event is emitted first. That could then overwrite the `context.emitted_cases` and there would be no need for this new step to check the logged events, it could use the existing step.